src/diagrams/git/gitGraphParserV2.spec.js

```diff
@@ -496,6 +496,76 @@ describe('when parsing a gitGraph', function () {
     ]);
   });
 
+  it('should handle merge ids', function () {
+    const str = `gitGraph:
+      commit
+      branch testBranch
+      checkout testBranch
+      commit
+      checkout main
+      %% Merge Tag and ID
+      merge testBranch tag: "merge-tag" id: "2-222"
+      branch testBranch2
+      checkout testBranch2
+      commit
+      checkout main
+      %% Merge ID and Tag (reverse order)
+      merge testBranch2 id: "4-444" tag: "merge-tag2"
+      branch testBranch3
+      checkout testBranch3
+      commit
+      checkout main
+      %% just Merge ID
+      merge testBranch3 id: "6-666"
```

Comment:
Is just Tag a valid scenario?

Reply:
Yep, it's in a previous test (added in https://github.com/mermaid-js/mermaid/pull/2955): https://github.com/mermaid-js/mermaid/blob/cde3a7cf70cbf6f33406ebb4e83e58d7af9feb7d/src/diagrams/git/gitGraphParserV2.spec.js#L464-L472 

Maybe it's worth potentially merging these tests together, instead of having a separate one just for merge ids.

Reply:
Cool!
More tests are fine. We don't know what edge case we might miss.